design/fast_finality.md

```diff
@@ -66,17 +72,17 @@ the client behave as if the transactions were completely final).
 
 ### Production of rollups
 
-Every `x` blocks, the sequencer's host requests the creation of a rollup. The sequencer's enclave produces a rollup 
-containing all the light batches created since the last rollup, in a sparse Merkle tree structure. This rollup is 
-encrypted and signed, then sent to be included on the L1.
-
 A rollup is produced whenever one of the following conditions is met:
 
-* The number of transactions across all light-batches since the last rollup exceeds `y`
-* The value of all transactions across all light-batches since the last rollup exceeds `z`
-* The time since the last rollup was produced exceeds `w`
+* The number of transactions across all light-batches since the last rollup exceeds `x`
+* The value of all transactions across all light-batches since the last rollup exceeds `y`
```

Comment:
instead of  "value" , it should be the "total size".
The reason for this rule, is that you have to fit the rollup in the ethereum gas limit.

Reply:
Addressed